README.md

```diff
@@ -42,6 +43,20 @@ remote Opam repository. A typical entry consists of three files:
 * `opam` - dependencies and build instructions
 * `url` - link to source code
 
+Note how versions are designated:
+
+    packages/xs/cdrom.0.9.2
+    packages/xs/crc.1.0.0
+    packages/xs-extra/libvirt.djs
+    packages/xs-extra/message-switch.master
+    packages/xs-extra/rrddump.master
+    packages/xs-extra/vncproxy.master
+    packages/xs-extra/wsproxy.master
+    packages/xs/fd-send-recv.1.0.2
+    packages/xs/nbd.2.1.2
```

Comment:
Maybe the `xs/` block could be before all the `xs-extra/` ones to make the message of this paragraph more obvious?
```
packages/xs/cdrom.0.9.2
packages/xs/crc.1.0.0
packages/xs/fd-send-recv.1.0.2
packages/xs/nbd.2.1.2
packages/xs/netlink.0.2.1
packages/xs-extra/libvirt.djs
packages/xs-extra/message-switch.master
packages/xs-extra/rrddump.master
packages/xs-extra/vncproxy.master
packages/xs-extra/wsproxy.master
```